src/main.rs

```diff
@@ -1,9 +1,9 @@
 use {
   crate::{arguments::Arguments, filter::Filter, state::State},
-  image::{ImageBuffer, RgbImage},
+  image::{io::Reader as ImageReader, ImageBuffer, RgbImage},
```

Comment:
I kind of dislike giving things custom names, since it makes them hard to look up and search.

```suggestion
  image::{ImageBuffer, RgbImage},
```